tests/lib/rules/await-async-query.test.ts

```diff
@@ -7,199 +8,194 @@ import {
 
 const ruleTester = createRuleTester();
 
+interface TestCode {
+  code: string;
+  isAsync?: boolean;
+}
+
+function createTestCode({ code, isAsync = true }: TestCode) {
+  return `
+    import { render } from '@testing-library/react'
+    test("An example test",${isAsync ? ' async ' : ' '}() => {
+      ${code}
```

Comment:
This function allows to get closer to a real test. I'm happy to move it to `test-utils` and refactor some tests (for example in `await-async-utils`) if needed 🙂

Reply:
This is interesting. I'd prefer to wait until v4 is released to avoid messing up the code and having to resolve lot of conflicts, but it's definitely a nice to have after it! It would need some improvements tho to be able to customize the import etc.